internal/types/binary.go

```diff
@@ -20,14 +20,29 @@ package types
 type BinarySubtype byte
 
 const (
-	BinaryGeneric    = BinarySubtype(0x00) // generic
-	BinaryFunction   = BinarySubtype(0x01) // function
+	// BinaryGeneric BSON generic binary subtype.
```

Comment:
There should be a verb like "represents"

Reply:
added.